manager/controlapi/service.go

```diff
@@ -302,11 +301,6 @@ func validateSecretRefsSpec(spec api.TaskSpec) error {
 		// If this is a file target, we will ensure filename uniqueness
 		if secretRef.GetFile() != nil {
 			fileName := secretRef.GetFile().Name
-			// Validate the file name
-			if fileName == "" || fileName != filepath.Base(filepath.Clean(fileName)) {
```

Comment:
Should this still return an error if the `fileName` is  empty, or are we allowing empty filenames and the orchestrator will just put it in `/run/secrets/<secret_name>`?  

If so, can we add/update the test for this case?